api/src/main/java/com/epam/pipeline/dao/user/UserDao.java

```diff
@@ -143,6 +147,33 @@ public PipelineUser updateUser(PipelineUser user) {
         return user;
     }
 
+    @Transactional(propagation = Propagation.MANDATORY)
+    public String createGroupBlockingStatus(String groupName, boolean blockStatus) {
+        int updatedRows = getNamedParameterJdbcTemplate().update(createGroupStatusQuery,
+                                                                 GroupParameters.getParameters(groupName, blockStatus));
+        return updatedRows > 0 ? groupName : null;
```

Comment:
Could you check group name existence before dao method? 
This way you may not to check updated rows count and make this method void. Also, it may be applied to update/delete methods

Reply:
Done